config/kafka-instance-types-configuration.yaml

```diff
@@ -46,6 +46,7 @@ supported_instance_types:
       capacityConsumed: 1
       supportedAZModes:
       - multi
+      maturityStatus: stable
```

Comment:
let's also make sure that https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/e6604e318d78ea293ca043bfc1ba9e16c69a1417/templates/service-template.yml#L160-L163 is updated.

Reply:
:+1: